src/wmnds/components/lists/unordered-list/_unordered-list.scss

```diff
@@ -0,0 +1,20 @@
+.wmnds-unordered-list, ul {
```

Comment:
You might have to remove the `ul` from this line

Reply:
Done!